open-requirements-library/agile-migrations.md

```diff
@@ -0,0 +1,845 @@
+### *Open Requirements*
+
+# Agile Migration to Drupal
```

Comment:
I think we need to add a license here - I would suggest some kind of owner contact details also (since readers may not know how to use Github)